datafn/python/datafn/filters.py

```
@@ -0,0 +1,35 @@
+from typing import Any, Dict, List
+
+def evaluate_filter(record: Dict[str, Any], filters: Dict[str, Any]) -> bool:
+    for key, value in filters.items():
+        if key == "$and":
+            if not isinstance(value, list): return False
```

Ruff (0.14.13)

Multiple statements on one line (colon)

(E701)